lua/headlines/init.lua

```diff
@@ -275,6 +278,14 @@ M.refresh = function()
                 })
             end
 
+            if capture == "list_marker" then
```

Comment:
I think you need to check if the user didn't disable these markers (similar to the other capture checks)
```suggestion
            if capture == "list_marker" and c.list_marker_string then
```